core/metacling/src/TCling.cxx

```diff
@@ -366,6 +366,16 @@ extern "C" void TCling__RestoreInterpreterMutex(void *delta)
    ((TCling*)gCling)->ApplyToInterpreterMutex(delta);
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Lookup libraries in LD_LIBRARY_PATH and DYLD_LIBRARY_PATH with mangled_name,
+/// which is extracted by error messages we get from callback from cling. Return true
+/// when the missing library was autoloaded.
+
+extern "C" bool TCling__LibraryLoadingFailed(const &std::string errmessage, const &std::string libStem, bool permanent, bool resolved)
```

Comment:
argg ... my bad ... I of course meant "const std::string& " .....

Reply:
Oh no sorry